src/radio-group/interfaces.ts

```diff
@@ -88,7 +88,12 @@ export namespace RadioGroupProps {
     value: string;
   }
 
-  export type Ref = RadioButtonProps.Ref;
+  export interface Ref {
```

Comment:
Duplicating this instead of importing from RadioButton because for some reason, the Typescript generation of type declarations would import the top-level RadioButtonProps import instead of the internal one, causing the Console build to break.